tensorflow/cc/gradients/nn_grad.cc

```diff
@@ -47,6 +47,81 @@ Status SoftmaxGrad(const Scope& scope, const Operation& op,
 }
 REGISTER_GRADIENT_OP("Softmax", SoftmaxGrad);
 
+bool IsZero(const Scope& scope, Output grad) {
+  std::array<std::string, 2> zeroOpTypeNames{{"ZerosLike", "Zeros"}};
+  string opTypeName = grad.op().node()->type_string();
+  for (auto& zeroOpTypeName : zeroOpTypeNames) {
+    if (opTypeName == zeroOpTypeName) {
```

Comment:
Please rename all variables names in the added code to be underscore separate lower case names like op_type_name.

https://google.github.io/styleguide/cppguide.html#Variable_Names